src/dhcp_service.rs

```
@@ -148,6 +162,13 @@ impl DhcpService {
             )),
         }
     }
+
+    fn release_v4_lease(&self, mut _client: DhcpV4Client) -> Result<(), DhcpServiceError> {
+        todo!()
```

Doesn't this just panic? Should we just return a normal error here instead?